docs/pitch/pitch-issue-83.html

```
@@ -1359,597 +1241,344 @@ <h2>Pluggable Schema Sources</h2>
+
+<span class="ky">steps</span>:
+  <span class="cm"># Each validate call auto-registers on success</span>
+  - <span class="ky">run</span>: <span class="vl">cvt validate --schema pet-api --schema-version 2.1.0 --interaction tests/get-pets.json</span>
+  - <span class="ky">run</span>: <span class="vl">cvt validate --schema pet-api --schema-version 2.1.0 --interaction tests/create-pet.json</span>
+  - <span class="ky">run</span>: <span class="vl">cvt validate --schema user-api --schema-version 1.0.0 --interaction tests/get-user.json</span></pre>
         </div>
       </div>
 
       <p class="speaker-note reveal d4">
-        The static check reads contract files and runs the CompatibilityEngine — no network, no server, no consumer CI triggered. Exit codes are stable so this drops straight into any CI pipeline.
+        The left terminal is what exists today. The right adds two flags. Nothing breaks, nothing changes for teams that don't use the registry. Environment variables avoid repetition in pipelines with multiple validate calls. Flag precedence: flag > env var > error.
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)